AxonDeepSeg/visualization/get_masks.py

```diff
@@ -18,9 +18,13 @@ def get_masks(path_prediction):
     
     # get main path
     path_folder, file_name = os.path.split(path_prediction)
+
     tmp_path = path_prediction.split('_seg-axonmyelin')
```

Comment:
This can be changed to: `tmp_path = path_prediction.split('_seg-axonmyelin')[0]`